packages/react/src/schematics/application/files/app/src/app/__fileName__.tsx__tmpl__

```diff
@@ -5,72 +5,218 @@ import React from 'react';
-  var header = 'Header';
-  var headerAttributes = '';
 %>import styled from '<%= styledModule %>';<% } else {
   var wrapper = 'div';
-  var header = 'header';
-  var headerAttributes = " style={{ textAlign: 'center'}}";
-%>import './app.<%= style %>';<% } %>
+%>import './<%= fileName %>.<%= style %>';<% } %>
 
 <% if (styledModule) { %>
 const StyledApp = styled.div`
-`;
+  /*
+   * Remove template code below
+   */
+
+  font-family: sans-serif;
+  min-width: 300px;
+  max-width: 600px;
+  margin: 50px auto;
+
+  .app-gutter-left {
+    margin-left: 9px;
+  }
+
+  .app-p {
+    text-align: center;
+  }
+
+  .app-col-span-2 {
+    grid-column: span 2;
+  }
+
+  .app-flex {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
 
-const Header = styled.header`
-  text-align: center;
+  .app-header {
+    background-color: #143055;
+    color: white;
+    padding: 5px;
+    border-radius: 3px;
+  }
+
+  .app-main {
+    padding: 0 36px;
+  }
+
+  .app-h1 {
+    text-align: center;
+    margin-left: 18px;
+    font-size: 24px;
+  }
+  .app-h2 {
+    text-align: center;
+    font-size: 20px;
+    margin: 40px 0 10px 0;
+  }
+
+  .app-resources {
+    text-align: center;
+    list-style: none;
+    padding: 0;
+    display: grid;
+    grid-gap: 9px;
+    grid-template-columns: 1fr 1fr;
+  }
+
+  .app-button {
+    color: #0094ba;
+    height: 36px;
+    background-color: rgba(0, 0, 0, 0);
+    border: 1px solid rgba(0, 0, 0, 0.12);
+    border-radius: 4px;
+    padding: 3px 9px;
+    text-decoration: none;
+  }
+
+  .app-button:hover {
+    background-color: rgba(68, 138, 255, 0.04);
+  }
+
+  .app-code {
+    padding: 9px;
+    border-radius: 4px;
+    background-color: black;
+    color: #eee;
+  }
+
+  .app-details {
+    border-radius: 4px;
+    color: #333;
+    background-color: rgba(0, 0, 0, 0);
+    border: 1px solid rgba(0, 0, 0, 0.12);
+    padding: 3px 9px;
+    margin-bottom: 9px;
+  }
+
+  .app-summary {
+    outline: none;
+    height: 36px;
+    line-height: 36px;
+  }
 `;
 <% }%>
 
+<%
+var innerJsx = `
+  <header className="app-header app-flex">
+    <img
+      alt=""
+      width="75"
+      src="https://nx.dev/assets/images/nx-logo-white.svg"
+    />
+    <h1 className="app-h1">Welcome to ${name}!</h1>
+  </header>
+  <main className="app-main">
+    <h2 className="app-h2">Resources &amp; Tools</h2>
+    <p className="app-p">
+      Thank you for using and showing some ♥ for Nx. <br />
+      Here are some links to help you get started.
+    </p>
+    <ul className="app-resources">
+      <li className="app-col-span-2">
+        <a
+          className="app-button app-flex"
+          href="https://nx.dev/react/getting-started/what-is-nx"
+        >
+          Nx video tutorial
+        </a>
+      </li>
+      <li className="app-col-span-2">
+        <a
+          className="app-button app-flex"
+          href="https://nx.dev/react/tutorial/01-create-application"
+        >
```

Comment:
I would remove the Angular Console link. We should only show it for Angular apps.